src/main/java/org/kohsuke/github/Requester.java

```diff
@@ -130,6 +130,10 @@ public Requester with(String key, int value) {
         return _with(key, value);
     }
 
+    public Requester with(String key, long value) {
+        return _with(key, value);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this line start at column 5. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)